libs/network/examples/CMakeLists.txt

```diff
@@ -22,7 +22,8 @@ if (FETCH_ENABLE_EXAMPLES)
   target_link_libraries(example-network-subscribe PRIVATE fetch-network)
   target_include_directories(example-network-subscribe PRIVATE subscribe)
 
-  add_executable(example-network-listener ../../ml/include/ml/ops/activations/gelu.hpp
+  add_executable(example-network-listener ../../ml/examples/bert_for_finetune/main.cpp
```

Comment:
this looks extremely wrong.
what does gelu and bert have to do with the networking code?